### DISCUSSION TITLE
Multi-Files Field - Target Folder

### DISCUSSION THREAD

**Discussion by ernst86913** [directus/directus#7609] [Archive]
Hi,
for Single file uploads we can specify the target folder upload path... how can this be done for the multi-file field? 

![image](https://user-images.githubusercontent.com/47870436/130847519-3bdba1e3-c3a6-49a7-8f6c-f32fc903a1a9.png)

Thx
Roman

**Comment by benhaynes** [comment]
This might not have been added yet. Certainly something we'd want to add though!

**Comment by ernst86913** [comment]
Ok - thanks - will work around it by uploading the files directly into the appropriate folder… and then add as links from there.
